spark/src/main/scala/org/apache/spark/sql/delta/OptimisticTransaction.scala

```diff
@@ -2317,4 +2317,59 @@ trait OptimisticTransactionImpl extends TransactionalWrite
       case _ =>
     }
   }
+
+  /**
+   * This method takes care of backfilling any unbackfilled delta files when managed commit is
+   * not enabled on the table (i.e. commit-owner is not present) but their are unbackfilled delta
+   * files in the table. This can happen if an error occurred during the MC -> FS commit where the
+   * commit-owner was able to register the downgrade commit but it failed to backfill it.
+   * This method must be invoked before doing the next commit as otherwise there will be a gap in
+   * the backfilled commit sequence.
+   */
+  def backfillWhenManagedCommitDisabled(): Unit = {
```

Comment:
This seems to only be called by the constructor?
```suggestion
  private def backfillWhenManagedCommitDisabled(): Unit = {
```

Reply:
done.